### PR TITLE
Update docs for creating a new Boefje

### LINKED ISSUE #3437: Update create boefje documentation
### Detailed description

We have to update this documentation https://docs.openkat.nl/developer_documentation/development_tutorial/creating_a_boefje.html with the new UI for creating boefjes.

### PR DESCRIPTION
**Pull request by madelondohmen**
### Changes
This PR adds some text to the `creating_a_boefje.md` documentation.

### Issue link

Closes #3437 

---

### Code Checklist

- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made. // Only changes to the documentation
- [x] I have checked the documentation and made changes where necessary.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: docs/source/developer_documentation/development_tutorial/creating_a_boefje.md [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+| Input object type | No       | Select the object type(s) that your Boefje. consumes.                                                                                                                                                                                                                                                                                   |
+| Output mime-types | No       | Add a set of mime-types that are produced by this Boefje, separated by commas. For example: _'text/html'_, _'image/jpeg'_ or _'boefje/{boefje-id}'_. These output mime-types will be shown on the Boefje detail page as information for other users.                                                                                    |
+| Clearance level   | No       | Select a clearance level for your Boefje, which indicates how deeply your Boefje can scan objects.                                                                                                                                                                                                                                      |
```
Comment (underdarknl) [inline docs/source/developer_documentation/development_tutorial/creating_a_boefje.md:25]:
```suggestion
| Clearance level   | No       | Select a clearance level for your Boefje, which indicates the clearance level the OOI's need before your Boefje will be scheduled on them.   |
```

File: docs/source/developer_documentation/development_tutorial/creating_a_boefje.md [hunk 2]
```diff
@@ -1,4 +1,38 @@
...
+| Arguments         | No       | For example: _'-sTU --top-ports 1000'_                                                                                                                                                                                                                                                                                                  |
+| Json Schema       | No       | If any other settings are needed for your Boefje, add these as a JSON Schema, otherwise, leave the field empty or 'null'. This JSON is used as the basis for a form for the user. When the user enables this Boefje they can get the option to give extra information. For example, it can contain an API key that the script requires. |
+| Input object type | No       | Select the object type(s) that your Boefje. consumes.                                                                                                                                                                                                                                                                                   |
```
Comment (ammar92) [inline docs/source/developer_documentation/development_tutorial/creating_a_boefje.md:23]:
```suggestion
| Input object type | No       | Select the object type(s) that your Boefje consumes.                                                                                                                                                                                                                                                                                   |
```

File: docs/source/developer_documentation/development_tutorial/creating_a_boefje.md [hunk 3]
```diff
@@ -1,4 +1,38 @@
...
+| ----------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Container image   | Yes      | The name of the Docker image. For example: _'ghcr.io/minvws/openkat/nmap'_                                                                                                                                                                                                                                                              |
+| Name              | Yes      | Give your Boefje a suiting name. This name will be visible in the KAT-alogus.                                                                                                                                                                                                                                                           |
```
Comment (ammar92) [inline docs/source/developer_documentation/development_tutorial/creating_a_boefje.md:19]:
```suggestion
| Name              | Yes      | Give your Boefje a suitable name. This name will be visible in the KAT-alogus.                                                                                                                                                                                                                                                           |
```

File: docs/source/developer_documentation/development_tutorial/creating_a_boefje.md [hunk 4]
```diff
@@ -1,4 +1,38 @@
...
+There are two ways to create a Boefje. The first is to create a Boefje with the User Interface (UI). The second option is to create it in the backend.
+
+## Creating a Boefje (variant) in the User Interface
+
+**Note:** Currently, only admins are able to create Boefjes in the UI.
+
+To create a **new** Boefje, go to the KAT-alogus. Here you will find the _'Add Boefje'_ button.
+To create a **variant** of an existing Boefje, go to the Boefje detail page of the Boefje you would like to use as a template and press the _'Add variant'_ button.
+
+### Setup your Boefje
```
Comment (ammar92) [inline docs/source/developer_documentation/development_tutorial/creating_a_boefje.md:12]:
```suggestion
### Set up your Boefje
```

**Review by ammar92 — COMMENTED** [review]
Looks good 👍 Just a few tiny remarks

**Review by underdarknl — APPROVED** [review]

**Pull request merged**